inc/classes/subscriber/Optimization/class-critical-css-subscriber.php

```diff
@@ -125,15 +172,15 @@ public function critical_css_generation_running_notice() {
 
 		$screen = get_current_screen();
 
-		if ( 'settings_page_wprocket' !== $screen->id ) {
-			return;
-		}
-
 		$transient = get_transient( 'rocket_critical_css_generation_process_running' );
 		if ( ! $transient ) {
 			return;
 		}
 
+		if ( 'settings_page_wprocket' !== $screen->id ) {
+		    return;
+		}
```

Comment:
Why change the position of this conditional check?

Reply:
Sorry, I copied it back in the wrong position while testing something. No need to be moved below get_transient();.